assets/js/pages/HostRelevantPatches/HostRelevantPatchesPage.jsx

```diff
@@ -32,12 +26,15 @@ function HostRelevantPatches({ hostName, onNavigate, patches }) {
   const [displayedPatches, setDisplayedPatches] = useState(patches);
 
   useEffect(() => {
-    setDisplayedPatches(
-      filterPatchesBySynopsis(
-        filterPatchesByAdvisoryType(patches, displayedAdvisories),
-        search
-      )
+    const filteredByAdvisoryKind = filterPatchesByAdvisoryType(
```

Comment:
```suggestion
    const filteredByAdvisoryType = filterPatchesByAdvisoryType(
```
?

Reply:
Just me thinking "wait this isn't a string or number... this is a kind". Changed it back... thanks for catching it :smile: